src/kitovu/sync/syncing.py

```diff
@@ -29,19 +30,23 @@ def _find_plugin(pluginname: str) -> syncplugin.AbstractSyncPlugin:
     plugin: syncplugin.AbstractSyncPlugin = manager.driver
     return plugin
 
-
```

Comment:
Unrelated whitespace change (two blank lines between top-level functions which aren't in a class)